module_auto_update/models/module.py

```diff
@@ -30,11 +30,15 @@ def _compute_checksum_dir(self):
         ).split(",")
 
         for r in self:
-            r.checksum_dir = dirhash(
-                get_module_path(r.name),
-                'sha1',
-                excluded_extensions=exclude,
-            )
+            try:
```

Comment:
OK, thanks for clarifying. Can you please put the same comment in code for later references?